src/Support/helpers.php

```diff
@@ -116,3 +116,16 @@ function dd(...$args)
         die(1);
     }
 }
+
+function inline($assetPath)
+{
+    preg_match('/^\/assets\/build\/(css|js)\/main\.(css|js)/', $assetPath, $matches);
```

Comment:
```suggestion
    preg_match('/^\/assets\/build\/(css|js)\/.*\.(css|js)/', $assetPath, $matches);
```
This is what worked for me locally

Reply:
Thanks @m1guelpf